include/anyode/anyode_blasless.hpp

```diff
@@ -1,123 +1,125 @@
 #pragma once
-#include <algorithm>  // std::min
```

Comment:
This was needed for MSVC when updating pyodeint or pygslodeiv2 to the latest AnyODE (std::min is used in the file).
(Ideally I guess this repo could use an Appveyor config at some point).

Reply:
Ah, good catch.